src/Microsoft.AspNetCore.SignalR.Core/IClientProxyExtensions.cs

```diff
@@ -0,0 +1,175 @@
+        /// <param name="arg3">The third argument</param>
+        /// <param name="arg4">The fourth argument</param>
+        /// <param name="arg5">The fifth argument</param>
+        /// <param name="arg6">The sixth argument</param>
+        /// <param name="arg7">The seventh argument</param>
+        /// <param name="arg8">The eigth argument</param>
+        /// <returns>A task that represents when the data has been sent to the client.</returns>
+        public static Task InvokeAsync(this IClientProxy clientProxy, string method, object arg1, object arg2, object arg3, object arg4, object arg5, object arg6, object arg7, object arg8)
+        {
+            return clientProxy.InvokeAsync(method, new object[] { arg1, arg2, arg3, arg4, arg5, arg6, arg7, arg8 });
+        }
+
+        /// <summary>
+        /// Invokes a method on the connection(s) represented by the <see cref="IClientProxy"/> instance.
+        /// </summary>
+        /// <param name="clientProxy">The <see cref="IClientProxy"/></param>
+        /// <param name="method">name of the method to invoke</param>
+        /// <param name="arg1">The first argument</param>
+        /// <param name="arg2">The second argument</param>
+        /// <param name="arg3">The third argument</param>
+        /// <param name="arg4">The fourth argument</param>
+        /// <param name="arg5">The fifth argument</param>
+        /// <param name="arg6">The sixth argument</param>
+        /// <param name="arg7">The seventh argument</param>
+        /// <param name="arg8">The eigth argument</param>
+        /// <param name="arg9">The ninth argument</param>
+        /// <returns>A task that represents when the data has been sent to the client.</returns>
+        public static Task InvokeAsync(this IClientProxy clientProxy, string method, object arg1, object arg2, object arg3, object arg4, object arg5, object arg6, object arg7, object arg8, object arg9)
+        {
+            return clientProxy.InvokeAsync(method, new object[] { arg1, arg2, arg3, arg4, arg5, arg6, arg7, arg8, arg9 });
+        }
+
+        /// <summary>
+        /// Invokes a method on the connection(s) represented by the <see cref="IClientProxy"/> instance.
+        /// </summary>
+        /// <param name="clientProxy">The <see cref="IClientProxy"/></param>
+        /// <param name="method">name of the method to invoke</param>
+        /// <param name="arg1">The first argument</param>
+        /// <param name="arg2">The second argument</param>
+        /// <param name="arg3">The third argument</param>
+        /// <param name="arg4">The fourth argument</param>
+        /// <param name="arg5">The fifth argument</param>
+        /// <param name="arg6">The sixth argument</param>
+        /// <param name="arg7">The seventh argument</param>
+        /// <param name="arg8">The eigth argument</param>
+        /// <param name="arg9">The ninth argument</param>
+        /// <param name="arg10">The ninth argument</param>
```

Comment:
This is the tenth argument :trollface:

Reply:
Also, these usually have a period at the end but I'll ignore them for this change...

Reply:
Good catch, I figure the copy pasta would catch up with me